src/common/platform/posix/unix/i_specialpaths.cpp

```diff
@@ -211,7 +212,13 @@ FString M_GetDocumentsPath()
 
 FString M_GetScreenshotsPath()
 {
+#ifdef __HAIKU__
 	return M_GetDocumentsPath() + "screenshots/";
+#else
+	static FString path = FStringf("%s/Screenshots/" GAMENAME, GetPicturesPath());
+	path = NicePath(path.GetChars());
+	return path;
+#endif
```

Comment:
```suggestion
#ifdef __HAIKU__
	static FString path = FStringf("%s/screenshots/" GAMENAMELOWERCASE, GetConfigPath());
#else
	static FString path = FStringf("%s/Screenshots/" GAMENAME, GetPicturesPath());
#endif
	path = NicePath(path.GetChars());
	return path;
```
This should set the screenshots directory on Haiku to  `~/config/settings/uzdoom/screenshots` as jscipione indicated.

Both directories are lowercase, for the UZDoom one it's because the .ini file creation already makes it like that, for "screenshots" it can be made uppercase no problem.

A similar thing should be done for the save directory but it's outside the scope of this pr.

Reply:
Updated (also savegame/demo dirs)

Reply:
Looks good to me :)